wmt-web/README.md

```diff
@@ -0,0 +1,16 @@
+# WMT Web
+This is the webserver application that will present the information stored in
+the WMT database to the end user.
+
+It is a node.js application using the express web framework.
+
+## Prerequisites
+- Node v7.6.x
```

Comment:
Might be worth checking this with Steven, but I have a feeling we were on the Node 6.9.0 (LTS) for the production APVS app, which would be recommendation here also.

We also should manage Node versions via NVM.